pkg/pci/json_test.go

```diff
@@ -1,1219 +0,0 @@
-// Copyright 2012-2017 the u-root Authors. All rights reserved
```

Comment:
Hmm I think this would be a good safety to keep. Why the removal?

Reply:
you can't easily parse the JSON any more, it will be a lot more complex. It's not really that important anyway.